reference-architecture/vmware-ansible/roles/haproxy-server/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
@sdodson @abutcher could either of you take a look at what it would take to be able to accomplish the same thing here with the haproxy role in openshift-ansible?